main/src/cgeo/geocaching/ui/CacheListAdapter.java

```diff
@@ -489,7 +489,8 @@ public void accept(final BitmapDrawable bitmapDrawable) {
             }
         }
 
-        holder.favorite.setText(Integer.toString(cache.getFavoritePoints()));
+        final int favCount = cache.getFavoritePoints();
+        holder.favorite.setText(favCount < 0 ? "?" : Integer.toString(favCount));
```

Comment:
```suggestion
        holder.favorite.setText(favCount >= 0 ? Integer.toString(favCount) : "?");
```

I think the inverted condition is more readable.

Reply:
Lint warning: Number formatting does not take into account local settings. Consider using `String.format` instead.

Reply:
Applied, and I'm not sure that we really care about number formatting here. Even if there are caches with more than 1k favs, I still find "1257" readable enough.